docs/getting_help.rst

```diff
@@ -0,0 +1,62 @@
+.. Copyright (C) 2023, Nansen Environmental and Remote Sensing Center
+
+.. raw:: html
+
+Getting Help
+============
+
+You can get help on compiling and running NextSIM-DG in several ways:
+
+- The documentation that can be found alongside this document.
+- The built-in help system.
+- By emailing the NextSIM-DG team.
```

Comment:
Provide email address here?

Reply:
@einola What email address would be best to use?

Reply:
I can ask IT to set up nextsim@nersc.no or something similar. Emails sent to there can then be forwarded to me and/or @timspainNERSC. How does that sound?

Reply:
That sounds just fine.

Reply:
Email address added.